corehq/apps/ota/views.py

```diff
@@ -184,7 +184,8 @@ def _add_case_property_queries(domain, case_type, search_es, criteria):
             case_property=key,
         )
         for removal_regex in remove_char_regexs:
-            value = re.sub(removal_regex.regex, '', value)
+            to_remove = re.escape(removal_regex.regex)
```

Comment:
are we sure that we want to escape all the characters in `removal_regex`? that would mean that we [lose](https://docs.python.org/2/library/re.html#re.escape) all pattern matching capabilities.

if `removal_regex` is configurable, i think it would be preferable to just configure the plus as `'\+` in the regex when trying to match an actual plus so that we can still use the `+` for pattern matching.